packages/typed-binary/src/io/bufferIOBase.ts

```diff
@@ -17,7 +17,8 @@ export class BufferIOBase {
   protected readonly uint8View: Uint8Array;
   protected readonly helperInt32View: Int32Array;
   protected readonly helperUint32View: Uint32Array;
-  protected readonly helperFloatView: Float32Array;
+  protected readonly helperFloat16View: Uint16Array;
```

Comment:
Since it is actually a uint16 view underneath, I would find this name confusing.
```suggestion
  protected readonly helperUint16View: Uint16Array;
```